CMakeLists.txt

```diff
@@ -90,10 +90,10 @@ ExternalBackendsInit()
 
 find_package(LLVM 8 CONFIG)
 if (NOT LLVM_FOUND)
-  # Fallback to LLVM 7
-  find_package(LLVM 7 CONFIG)
+  # Fallback to whatever is available.
+  find_package(LLVM CONFIG)
 
-  if (NOT LLVM_FOUND)
+  if (NOT LLVM_FOUND OR LLVM_VERSION VERSION_LESS "7.0")
     message(FATAL_ERROR "LLVM minimum required version is 7. LLVM package version found: ${LLVM_PACKAGE_VERSION}.")
```

Comment:
Intentional to have LLVM_VERSION in the if(...) and LLVM_PACKAGE_VERSION in the message?